packages/discord.js/src/structures/Poll.js

```diff
@@ -48,9 +69,9 @@ class Poll extends Base {
 
     /**
      * The timestamp when this poll expires
-     * @type {number}
+     * @type {?number}
      */
-    this.expiresTimestamp = Date.parse(data.expiry);
+    this.expiresTimestamp = Date.parse(data.expiry) ?? null;
```

Comment:
This change here is to prevent a hard crash once polls without expiry dates roll out, and it is also marked as nullish on the API documentation